pipelines/batch/setup_eval_job.py

```diff
@@ -158,12 +165,17 @@ def main(
     for disease, report_date, loc in itertools.product(
         disease_list, report_dates, all_locations
     ):
+        n_training = 90
+        exclude_last_n = 3
         task = get_task_config(
             f"{job_id}-{loc}-{disease}-{report_date}",
             base_call=base_call.format(
                 state=loc,
                 disease=disease,
                 report_date=report_date,
+                n_training=n_training,
+                exclude_last_n=exclude_last_n,
+                output_dir=str(Path("output", output_subdir)),
```

Comment:
Should the "output" part be configurable?

Reply:
No, because that's the hard-coded bind mount location of the output blob storage container within the running podman/docker container.

Reply:
See https://github.com/CDCgov/pyrenew-hew/pull/159/files/a2fa9afa9651fda572c50b4a2db90b8379dc468b#r1859432678